code/modules/clothing/under/syndicate.dm

```diff
@@ -59,6 +59,12 @@
 	has_sensor = TRUE
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0, "wound" = 5)
 
+/obj/item/clothing/under/syndicate/cosmetic/skirt
+	name = "tacticool skirtleneck"
```

Comment:
tactitool for consistency

Reply:
The name doesn't have to do with anything look at the one above.

Reply:
oh wait, now i see what i did wrong